prefect_aws/glue_job.py

```diff
@@ -0,0 +1,192 @@
+"""
+Integrations with the AWS Glue Job.
+
+"""
+import time
+from typing import Any, Optional
+
+from prefect.blocks.abstract import JobBlock, JobRun
+from prefect.utilities.asyncutils import run_sync_in_worker_thread
+from pydantic import VERSION as PYDANTIC_VERSION
+
+if PYDANTIC_VERSION.startswith("2."):
+    from pydantic.v1 import BaseModel, Field
+else:
+    from pydantic import BaseModel, Field
+
+from prefect_aws import AwsCredentials
+
+
+class GlueJobRun(JobRun, BaseModel):
```

Comment:
This class doesn't need to be a Pydantic BaseModel since we don't need validation. I think it might be simpler to make it a plain Python class

```suggestion
class GlueJobRun(JobRun):
```

Reply:
I find it useful to depend on pydantic for uniformity with other object definitions, validation, and attribute descriptions.